components/config/src/config.rs

```diff
@@ -154,6 +172,8 @@ pub struct Config {
     /// Which themes to use for code highlighting. See Readme for supported themes
     /// Defaults to "base16-ocean-dark"
     pub highlight_theme: String,
+    /// Generate CSS files for Thmes out of syntect
```

Comment:
themes

Reply:
Also renamed the Vector to `generate_themes_css`.